src/jsdoc_transformer.ts

```diff
@@ -270,16 +270,22 @@ function createMemberTypeDeclaration(
           p, `Skipping unhandled member: ${escapeForComment(p.getText())}`)));
 
   for (const fnDecl of abstractMethods) {
-    const name = propertyName(fnDecl);
+    // If the function declaration is computed, its name is the computed expression; otherwise, its
+    // name can be resolved to a string.
+    const name = fnDecl.name && ts.isComputedPropertyName(fnDecl.name) ? fnDecl.name.expression :
+                                                                         propertyName(fnDecl);
     if (!name) {
       mtt.error(fnDecl, 'anonymous abstract function');
       continue;
     }
     const {tags, parameterNames} = mtt.getFunctionTypeJSDoc([fnDecl], []);
     if (hasExportingDecorator(fnDecl, mtt.typeChecker)) tags.push({tagName: 'export'});
+    // Element access for computed names.
```

Comment:
This was hard to follow because of no verb, maybe something like "Use element access instead of property access for computed names" or so?

Reply:
Fixed.